source/math/FloatTransform3D.ooc

```diff
@@ -208,7 +208,7 @@ FloatTransform3D: cover {
 	}
 	transformAndProject: func ~FloatPoint2D (point: FloatPoint2D, focalLength: Float) -> FloatPoint2D {
 		transformedWorldPoint := this * FloatPoint3D new(point x, point y, focalLength)
-		this project(transformedWorldPoint, focalLength)
+		focalLength == 0.0f ? FloatPoint2D new(transformedWorldPoint x, transformedWorldPoint y) : this project(transformedWorldPoint, focalLength)
```

Comment:
No, we should avoid having to check for `NaN`, of course, but when we do, it would be nice to have a property for it. See https://github.com/cogneco/ooc-kean/issues/626.